rs/ic-observability/log-noise-filter-backend/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "log-noise-filter-backend"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+description.workspace = true
+documentation.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+clap = { workspace = true }
+axum = "0.7.4"
```

Comment:
Should we move this to the workspace? We seem to be using axum in a few places now.